x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/OperatorTests.java

```diff
@@ -324,6 +328,72 @@ public ScoreMode scoreMode() {
         return docIds;
     }
 
+    public void testHashLookup() {
```

Comment:
I've not plugged the `HashLoopupOperator` into anything yet so good old `OperatorTests` will have to do. This has been here since long before I started working on ESQL and it's kind of a junk drawer. But it'll do until we get this wired in.